src/azul/service/elasticsearch_service.py

```diff
@@ -586,6 +589,18 @@ def transform_request(self,
                 es_response = es_search.execute(ignore_cache=True)
             except elasticsearch.NotFoundError as e:
                 raise IndexNotFoundError(e.info["error"]["index"])
+            except elasticsearch.RequestError as e:
+                if one(e.info['error']['root_cause'])['reason'].startswith('No mapping found for'):
+                    es_response = self.es_client.count(index=config.es_index_name(catalog=catalog,
+                                                                                  entity_type=entity_type,
+                                                                                  aggregate=True))
+                    if not es_response['count']:  # Count is zero for empty index
+                        final_response = FileSearchResponse(hits={},
```

Comment:
reassign to es_response

Reply:
I was using precedence for the assignation, as used in the non empty response.

Reply:
I don't understand.

Reply:
At the end of `transform_request` the variable `final_response` is used to store the outgoing request's response. I was using that as precedence for naming the variable you requested I change.
It feels odd to use `es_response` to store the request's response. I think it is because `es_response` is used internally to store the response coming from Elasticsearch, which isn't representative of an outgoing response. But if you prefer it that way, I have no objection.

Reply:
@hannes-ucsc: "I now understand, go back to `final_response`."